server/chat-plugins/hangman.ts

```diff
@@ -299,6 +327,86 @@ export const commands: ChatCommands = {
 		''(target, room, user) {
 			return this.parse('/help hangman');
 		},
+
+		random(target, room, user) {
+			room = this.requireRoom();
+			this.checkCan('mute', null, room);
+			if (room.game) {
+				throw new Chat.ErrorMessage(`There is already a game of ${room.game.title} running.`);
+			}
+			const {question, hint} = Hangman.getRandom(room.roomid);
+			const game = new Hangman(room, user, question, hint);
+			room.game = game;
+			this.addModAction(`${user.name} started a random game of hangman - use /guess to play!`);
+			game.display(user, true);
+			this.modlog(`HANGMAN RANDOM`);
+		},
+		addrandom(target, room, user) {
+			room = this.requireRoom();
+			this.checkCan('mute', null, room);
+			if (!hangmanData[room.roomid]) hangmanData[room.roomid] = {};
+			if (!target) return this.parse('/help hangman');
+			// validation
+			const args = target.split(',');
```

Comment:
```suggestion
			const args = target.split('|');
```

some people might want hints with commas in them